pkg/ccl/backupccl/show.go

```diff
@@ -670,9 +672,10 @@ func backupShowerDefault(
 				// Map database ID to descriptor name.
 				dbIDToName := make(map[descpb.ID]string)
 				schemaIDToName := make(map[descpb.ID]string)
+				typeIDToTypeDescriptor := make(map[descpb.ID]catalog.TypeDescriptor)
 				schemaIDToName[keys.PublicSchemaIDForBackup] = catconstants.PublicSchemaName
 				for i := range manifest.Descriptors {
```

Comment:
I think the types returned from each function is different. If I used a switch I think I would have to call a new builder in each condition of the switch instead of the unioned struct already provided in `FromDescriptor`